### PR TITLE
Create macro to add prefix BELG to headline. [SDBELGA-196]

### PR DESCRIPTION
**Pull request by GyanP**
belg-headline

### CODE REVIEW

File: server/belga/macros/__init__.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Use `superdesk.macro_register.macros.register` for registration.
+"""
+
+import os
+import sys
+import imp
+import importlib
+
+
+def load_macros(path):
```
Comment (petrjasek) [inline server/belga/macros/__init__.py]: I think you can just import it:
```
import belga.macros.add_prefix_to_headline
```

Comment (GyanP) [reply server/belga/macros/__init__.py]: I thought let's keep it consistent with other repo.

File: server/belga/macros/__init__.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"""Here you can import/implement macros.
+
+This module will get reloaded per request to be up to date.
+
+Use `superdesk.macro_register.macros.register` for registration.
+"""
```
Comment (petrjasek) [inline server/belga/macros/__init__.py]: I think this comment is off now

**Comment by GyanP** [comment]
@petrjasek all other macros from the core like `POPULATE ABSTRACT`, `VALIDATE FOR PUBLISH` etc. are not available anymore. Is that ok?

**Comment by petrjasek** [comment]
@karelpetrak do they need any core macros?

**Comment by GyanP** [comment]
All of them will be available now, as Jerome made some changes.

**Pull request merged**